backend/api/battles/serializers.py

```diff
@@ -57,7 +79,57 @@ def validate(self, attrs):
         if pokemon_points_sum > 600:
             raise serializers.ValidationError("Pokemons' points sum cannot be more than 600")
 
+        if self.instance:
+            object_id = self.instance.id
+
+            battle = Battle.objects.get(pk=object_id)
```

Comment:
Why do you need to fetch the battle again instead of just using the `self.instance`?

Reply:
You're right. That doesn't make sense. I forgot I already have the `self.instance`.